assets/scss/theme.scss

```diff
@@ -67,3 +67,8 @@ $px-20: 1.2500rem;
 $px-24: 1.5000rem;
 $px-32: 2.0000rem;
 $px-40: 2.5000rem;
+
+// Font colors
+$dk-text: $gray9;
+$text: $gray7;
```

Comment:
Why the alias and not just use `$gray9` or `$gray7`. Or is this overriding these classes locally?

Reply:
I copied it from namerequest.

Reply:
hmm, well, i'm not sure the scope of fixing but if easily fixed, i would just use the theme colours `$gray9` or `$gray7`.
Im not sure the purpose of the alias. 
No biggie either way